astropy/visualization/wcsaxes/wcsapi.py

```diff
@@ -0,0 +1,254 @@
+# Functions/classes for WCSAxes related to APE14 WCSes
+
+import numpy as np
+
+from astropy.coordinates import SkyCoord, ICRS
+from astropy import units as u
+from astropy.wcs import WCS
+from astropy.wcs.wcsapi import SlicedLowLevelWCS
+
+from .frame import RectangularFrame, EllipticalFrame
+from .transforms import CurvedTransform
+
+__all__ = ['transform_coord_meta_from_wcs', 'WCSWorld2PixelTransform',
+           'WCSPixel2WorldTransform']
+
+IDENTITY = WCS(naxis=2)
```

Comment:
Looks like we can delete this? We aren't using it in this file, or are we importing it elsewhere?

Reply:
Fixes